arsceneview/src/main/java/io/github/sceneview/ar/ARSceneView.kt

```diff
@@ -565,8 +562,6 @@ open class ARSceneView @JvmOverloads constructor(
 
     override fun destroy() {
         if (!isDestroyed) {
-            destroyARCore()
```

Comment:
You use LifeCycleObserver as the main place to call ArCore lifecycle functions
And you don't need to call it here again
I didn't find any side cases